pkgs/top-level/python-packages.nix

```diff
@@ -5762,6 +5762,8 @@ in {
       inherit (pkgs) pkgconfig;
     };
 
+  python-nest = callPackage ../development/python-modules/python-nest { };
```

Comment:
This should probably be above pythonnet, no?

Reply:
Right you are :)